packages/lab/src/splitter/SplitHandle.tsx

```diff
@@ -0,0 +1,86 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import { useContext } from "react";
+import {
+  PanelResizeHandle,
+  type PanelResizeHandleProps,
+} from "react-resizable-panels";
+
+import splitHandleCSS from "./SplitHandle.css";
+import { AppearanceContext, OrientationContext } from "./Splitter";
+import { computeAccent, computeVariant } from "./utils";
+
+const withBaseName = makePrefixer("saltSplitHandle");
+
+export type SplitHandleVariant =
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "transparent";
+
+export type SplitHandleAccent =
+  | "top"
+  | "bottom"
+  | "right"
+  | "left"
+  | "top-bottom"
+  | "left-right"
+  | "none";
+
+export interface SplitHandleProps extends PanelResizeHandleProps {
+  /**
+   * Styling variant
+   * @default "primary"
+   */
+  variant?: SplitHandleVariant;
+  /**
+   * Change which sides get a border displayed
+   *
+   * Default is based on the orientation and appearance
+   * set on the parent Stepper components, ex.
+   * bordered + horizontal = left-right
+   * bordered + vertical = top-bottom
+   * transparent = none
+   */
+  accent?: SplitHandleAccent;
```

Comment:
Is accent the right terminology? cc @ivan-calderon

Reply:
Changed to `border` as agreed